test/e2e/common/node/pods.go

```diff
@@ -1042,7 +1042,7 @@ var _ = SIGDescribe("Pods", func() {
 		framework.ExpectNoError(err, "failed to delete Pod by collection")
 
 		ginkgo.By("watching for the Pod to be deleted")
-		ctxUntil, cancel = context.WithTimeout(ctx, 1*time.Minute)
```

Comment:
@Riaankl  @dashpole Do you have the context that why we use 1 minute for a pod deletion here?

https://github.com/kubernetes/kubernetes/blob/dc3f5ec6ccb9855dfa99f4c1078625df5fdfab6a/test/e2e/framework/timeouts.go#L26-L27

We meet some flake in https://testgrid.k8s.io/sig-release-master-informing#periodic-kubernetes-e2e-kind-kms https://github.com/kubernetes/kubernetes/issues/108891, so I change it to 5 minute which is the default Pod deletion timeout.

Reply:
it seems is an aleatory number https://github.com/kubernetes/kubernetes/commit/079764ebd5477c77acb11a55587d2499f9c25138 , we should use the common timeouts for consistency

/lgtm
/approve